### ISSUE TITLE
firing up a host gives me an SSH error

### ISSUE THREAD

**Issue by EugenMayer** [mxschmitt/ui-driver-hetzner#5]
The first steps are working just fine, but in the end, the SSH connection does not worked.

Deployed a C21 on Falkenstein. Any hints?

**Comment by mxschmitt** [comment]
Which image, size and rancher version?
Can you reproduce it?

**Comment by EugenMayer** [comment]
i can 100% reproduce it.

image was debian 9, rancher is 1.6.13 with the newest rancheros 1.2.0 as the host.
On question, for the driver itself - how is the SSH key distributed in general?

**Comment by mxschmitt** [comment]
I tried to reproduce it using:
- Rancher: v1.6.14
- Cattle: v0.183.37
- User Interface: v1.6.37
- Rancher CLI: v0.6.7
- Rancher Compose: v0.12.5

and the Host:
- Debian 9.3
- CX21
- Falkenstein

Rancher is running on debian 9.3 also, but I think this won't cause the issue. It works successfully, without any issues, could you provide me any logs from the rancher container stdout/stderr? There should be the command attached which fails.

The handling / ssh key exchange isn't part of the ui driver. This will be done in the docker machine driver [here](https://github.com/JonasProgrammer/docker-machine-driver-hetzner). Maybe an upstream issue. You could try to create a host manually by the command line instead of using the rancher-ui-driver. (But be sure, that you use overlay2 as storage provider).

**Comment by EugenMayer** [comment]
@mxschmitt i am actually tracing this back to probably something else. I firewalled my rancher server :8080 to only be accessable by my rancher-hosts. Well that was obviously a new rancher host and a new IP and most probably could not reach :8080 leading to this kind of issue probably.

I did not verify this yet, its not as easy to do since i cannot really unlock an ip prior the setup, since its not known. But i guess i can disable the firewall and try again.

Sorry for creating this kind of effort on your side, especially if this comes down to an infrastructural aspect on my side. Let me test that stuff and wait for my finding before you dig any deeper
